protocol/ping/ping.go

```diff
@@ -0,0 +1,64 @@
+package ping
+
+import (
+	"errors"
+	"github.com/kaspanet/kaspad/netadapter/router"
+	peerpkg "github.com/kaspanet/kaspad/protocol/peer"
+	"github.com/kaspanet/kaspad/util/random"
+	"github.com/kaspanet/kaspad/wire"
+	"time"
+)
+
+const pingInterval = 2 * time.Minute
+
+// HandlePing handles all ping messages coming through incomingRoute.
```

Comment:
The names `HandlePing` and `StartPingLoop` are a bit confusing, especially that both of them are loops.

I'd rename to `HandleIncomingPings` and `StartSendPingsLoop` or something similar (I don't 100% love `StartSendPingsLoop` but that's the best I was able to come up with)